constant_editor.go

```diff
@@ -70,22 +70,36 @@ func (m *Manager) editConstants() error {
 				return fmt.Errorf("couldn't edit %s in %v: %w", constantEditor.Name, id, err)
 			}
 		}
+	}
+
+	// Apply to all programs if no section was provided
+	for section, prog := range m.collectionSpec.Programs {
+		edit := newEditor(&prog.Instructions)
```

Comment:
Should we wait to allocate this until we know there are constant editors that apply to all programs?

Reply:
I think lazy-allocation would be sufficient, waiting until both `continue` statements have been passed.

Reply:
That's a good point, should be good in the latest commit